tests/test_clan_members_rank.py

```diff
@@ -47,16 +47,73 @@ def test_compare_card_levels(self):
 
     def test_get_card_level_counts(self):
         # given
-        expected_card_level_counts = {1: 1, 2: 0, 3: 0, 4: 0, 5: 1, 6: 1, 7: 1, 8: 2, 9: 6, 10: 27, 11: 28, 12: 24, 13: 12}
-        player_cards = clash_royale_client_responses.PLAYER_2_RESPONSE['cards']
+        expected_card_level_counts = {
+            1: 1, 2: 0, 3: 0, 4: 0, 5: 1, 6: 1, 7: 1, 8: 2, 9: 6, 10: 27, 11: 28, 12: 24, 13: 12}
+        filtered_troop_expected_card_level_counts = {
+            1: 1, 2: 0, 3: 0, 4: 0, 5: 1, 6: 0, 7: 1, 8: 1, 9: 4, 10: 17, 11: 19, 12: 18, 13: 9}
+        filtered_spell_expected_card_level_counts = {
+            1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 0, 7: 0, 8: 1, 9: 1, 10: 4, 11: 7, 12: 4, 13: 2}
+        filtered_building_expected_card_level_counts = {
+            1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 1, 7: 0, 8: 0, 9: 1, 10: 6, 11: 2, 12: 2, 13: 1}
 
+        filtered_troop_expected_card_level_counts_p1 = {1: 1, 2: 0, 3: 0, 4: 1, 5: 0,

```

Comment:
/supernit, why is this formatting so aids? hahahaha can't you make it consistent with the above or is your linter doing this? 
i.e. make it something like this
```
        expected_card_level_counts = {
            1: 1, 2: 0, 3: 0, 4: 0, 5: 1, 6: 1, 7: 1, 8: 2, 9: 6, 10: 27, 11: 28, 12: 24, 13: 12}
```
Instead of
```
        expected_card_level_counts = {1: 1, 2: 0, 3: 0, 4: 0, 5: 1, 
              6: 1, 7: 1, 8: 2, 9: 6, 10: 27, 11: 28, 12: 24, 13: 12}
```

Reply:
done